src/integrationTest/java/uk/gov/hmcts/reform/civil/controllers/cases/CasesControllerTest.java

```diff
@@ -34,6 +39,24 @@ public class CasesControllerTest extends BaseIntegrationTest {
         + "\n"
         + " }\n"
         + "}";
+    private static final String CLAIMANT_CLAIMS_URL = "/cases//claimant/{submitterId}";
```

Comment:
should be `/cases/claimant/{submitterId`?

Reply:
done 👍